Sources/MapboxMaps/Gestures/GestureHandlers/GestureHandler.swift

```diff
@@ -2,6 +2,10 @@ import UIKit
 
 internal protocol GestureHandlerDelegate: AnyObject {
     func gestureBegan(for gestureType: GestureType)
+
+    func gestureEnded(for gestureType: GestureType, willDecelerate: Bool)
+
+    func driftEnded(for gestureType: GestureType)
```

Comment:
Let's use the word deceleration instead of drift for consistency

Reply:
Superseded by https://github.com/mapbox/mapbox-maps-ios/pull/697#discussion_r714753031